### PR TITLE
Adjustment of some Russian translations

### PR DESCRIPTION
**Pull request by DiMan**
As written in PR #204, here are my changes:

`msgfmt -c --statistics ru.po`
4672 translated messages, 8 fuzzy translations, 5 untranslated messages.

`msgfmt -c --check-accelerators="_" --statistics ru.po`
71 errors
@all: Changes for keyboard accelerators I will send separately.

### CODE REVIEW

**Comment by DiMan** [comment]
Thanks!

Am 11.10.2017 11:15 nachm. schrieb "GnuCash GitHub Account" <
notifications@github.com>:

**Comment by DiMan** [comment]
Thanks!

Am 11.10.2017 11:15 nachm. schrieb "GnuCash GitHub Account" <
notifications@github.com>:

Merged #211 <https://github.com/Gnucash/gnucash/pull/211>.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/Gnucash/gnucash/pull/211#event-1289087514>, or mute the
thread

File: po/ru.po [hunk 1]
```diff
@@ -1199,7 +1199,7 @@ msgid ""
 "This copy was built from %s rev %s on %s."
 msgstr ""
 "%s\n"
-"Эта копия собрана из %s, ревизия %s. Дата сборки %s."
+"Эта копия собрана из %s, ревизия %s. \nДата сборки %s."
```
Comment (fellen) [inline po/ru.po]: It is better to insert only layout elements, which are already in the msgid. You don't know the size of the displaying window.

**Comment by DiMan** [comment]
Hello Frank,

I checked it in ubuntu. It looked with the line break much better. Do you
think it can make problems in the display on other operating systems, like
Windows?

Best Regards, Dmitriy


Am 11.10.2017 11:24 nachm. schrieb "Frank H. Ellenberger" <
notifications@github.com>:

*@fellen* commented on this pull request.

Thanks!
------------------------------

In po/ru.po
<https://github.com/Gnucash/gnucash/pull/211#discussion_r144140045>:

> @@ -1199,7 +1199,7 @@ msgid ""
 "This copy was built from %s rev %s on %s."
 msgstr ""
 "%s\n"
-"Эта копия собрана из %s, ревизия %s. Дата сборки %s."
+"Эта копия собрана из %s, ревизия %s. \nДата сборки %s."

It is better to insert only layout elements, which are already in the
msgid. You don't know the size of the displaying window.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/Gnucash/gnucash/pull/211#pullrequestreview-68756576>,
or mute the thread

**Comment by fellen** [comment]
> I checked it in ubuntu. 
gnucash --version
or the about box?
> it can make problems to display it on other operating systems
I don't know and can not test all.

Perhaps you should add a 
#  translator-comment
line, in case later someone complains.

**Comment by DiMan** [comment]
to check it, I built new stable version from source. On Ubuntu 17.04. 
This text is displayed in the "about" window.

you are right. I will add translator-commit for this line.

**Comment by fellen** [comment]
It is used twice:
ru.po:1195 `#: ../src/bin/gnucash-bin.c:452 ../src/gnome-utils/gnc-main-window.c:4386`
gnucash-bin.c:452 is part of `gnucash --version`
gnc-main-window.c:4386 of the About dialog.

**Comment by DiMan** [comment]
Hello Frank,

I corrected it to make it look better. In both cases, it looks better with a line break:

`gnc-main-window.c:4386` - part of the About dialog:

[](https://postimg.org/image/1lgyyqtkuz/)

`gnucash-bin.c:452` - part of `gnucash --version`:

[](https://postimg.org/image/4p4nytjqmz/)

I would like to add translator-comments. Should I send new PR or can I change this one?

Does it make sense to transfer the changes with the line break to the English and German versions?

**Comment by DiMan** [comment]
For example "about dialog" without line breaks in English and German:

[](https://postimg.org/image/8gynm7hqa3/)

[](https://postimg.org/image/95rx68g6t7/)

**Comment by fellen** [comment]
> Does it make sense to transfer the changes with the line break to the English and German versions?

Formally "Дата сборки 2017-10-07." is no full sentence. So I personally prefer the original form:
```
> LANG=C gnucash --version
GnuCash 2.6.18 development version
This copy was built from git rev f283437+ on 2017-10-11.
> gnucash --version
GnuCash Entwicklungsversion 2.6.18
Dieses Programm wurde aus git Version f283437+ am 2017-10-11 erstellt.
```
But feel free to ask on gnucash-devel for other opinions, linking this thread.
> Should I send new PR or can I change this one?

Because this PR is already merged, you should create a new one.  BTW, I know at least 1 changed string, so you should https://wiki.gnucash.org/wiki/Translation#Updating_an_existing_.po_file again.

**Comment by DiMan** [comment]
Hello Frank,

Sorry, I don't quite exactly put it. I have meant the line break for the following sentence:

`#: ../src/gnome-utils/gnc-main-window.c:4370`
msgid "The GnuCash personal finance manager. **\n**The GNU way to manage your money!"
msgstr "GnuCash: Ihr privater Finanzmanager. **\n**Die freie Lösung zur Finanzverwaltung."

Change for only first two sentences in the about dialog.

For English version:
```
The GnuCash personal finance manager. The GNU way to 
manage your money! 
```
--> 
```
The GnuCash personal finance manager. 
The GNU way to manage your money! 
```

and for German version:
```
GnuCash: Ihr privater Finanzmanager. Die freie Lösung 
zur Finanzverwaltung.
```
--> 
```
GnuCash: Ihr privater Finanzmanager. 
Die freie Lösung zur Finanzverwaltung.
```

**Comment by gjanssens** [comment]
I'm fine with adding a newline in the gnucash tag line in the original English version as well. If you want to go ahead, just make a new PR for it. Note the line describing the git commit that was used for the build has changed in the unstable series. In one of your screenshots above you had changed that line as well. I'd propose to leave it alone in maint at least to avoid additional merge conflicts.

**Comment by DiMan** [comment]
Hello fellen and gjanssens,
I just submitted a new PR for it (#218).

**Pull request merged**